java/ql/src/Security/CWE/CWE-276/ContentProviderIncompletePermissions.qhelp

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>In the Android manifest file, an application's <code>provider</code> elements
+  define the permissions necessary to access a resource using that provider.
+  Permissions are specified with
+  the <code>android:readPermission</code>, <code>android:writePermission</code>,
+  or <code>android:permission</code> attributes. If an application only
+  specifies the <code>android:readPermission</code>
+  or <code>android:writePermission</code> attribute, no permissions will be
+  required to do other operations.
+</p>
+
+<p>Content providers should either define both the read and write permissions
+attributes, or define the general <code>android:permission</code> attribute.</p>
```

Comment:
Could the last sentence of the overview potentially be removed since it repeats in the recommendation section?
```suggestion
```